src/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ file(GLOB
     *.cc
     ${PROJECT_SOURCE_DIR}/include/benchmark/*.h
     ${CMAKE_CURRENT_SOURCE_DIR}/*.h)
-list(FILTER SOURCE_FILES EXCLUDE REGEX "benchmark_main\\.cc")
+list(REMOVE_ITEM SOURCE_FILES "benchmark_main.cc")
```

Comment:
Does that actually work though?
`GLOB` should result in absolute filenames..

Reply:
Right, fix coming

Reply:
Switched to solution suggested by @kfitch in #603